src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/ContentHelper.Common.cs

```diff
@@ -21,8 +21,15 @@ internal static class ContentHelper
         // ContentType may not exist in response header.  Return null if not.
         internal static string? GetContentType(HttpResponseMessage response) => response.Content.Headers.ContentType?.MediaType;
 
+        internal static string? GetContentType(HttpRequestMessage request) => request.Content?.Headers.ContentType?.MediaType;
+
         internal static Encoding GetDefaultEncoding() => Encoding.UTF8;
 
+        internal static string GetFriendlyContentLength(long? length) =>
+            length.HasValue
+            ? $"{length.Value.Bytes().Humanize()} ({length.Value.ToString("#,0").Replace(",", " ")} bytes)"
```

Comment:
I guess Humanize uses current culture. So ToString should do the same.
```suggestion
            ? $"{length.Value.Bytes().Humanize()} ({length.Value.ToString("#,0")} bytes)"
```